docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
+- [ ] Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
+- [ ] Enhance BIS schemas for common IOT patterns and query engines.
 
 ### Agent Deployment
 
-- Provide working examples and templates of agents deployed via Kubernetes.
-- Perform operations on iModels that are triggered by relevant events from iModelHub.
+- [ ] Provide working examples and templates of agents deployed via Kubernetes.
+- [ ] Perform operations on iModels that are triggered by relevant events from iModelHub.
 
 ### iTwin Viewer
 
-- Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
-- Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
+- [ ] Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
+- [ ] Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
 
 ### Point Clouds
 
-- Support visualization of time-based differencing of large scale point clouds.
+- [ ] Support visualization of time-based differencing of large scale point clouds.
 
 ### Reality Data Sources
 
-- Implement a grid of linear transformations for large reality data.
+- [ ] Implement a grid of linear transformations for large reality data.
 
 ### Geolocated Photos and Panoramas
 
-- Improve server-side extraction of tags from photos and panoramas.
-- Incorporate geophoto package into an Extension loadable from any iTwin Viewer application.
+- [ ] Improve server-side extraction of tags from photos and panoramas.
+- [ ] Incorporate geophoto package into an Extension loadable from any iTwin Viewer application.
 
 ### Geolocation (general)
 
-- Provide a method to access geolocation gcs definition for i model and context data horizontal and vertical
-- Implement frontend reprojection when possible based on orbit gt implementation.
-- Implement backend of new geo coord service reprojection for complex gcs.
-- Obtain geoid separation from new geo coord service.
+- [X] Provide a method to access geolocation gcs definition for i model and context data horizontal and vertical
+- [ ] Implement frontend reprojection when possible based on orbit gt implementation.
+- [ ] Implement backend of new geo coord service reprojection for complex gcs.
+- [ ] Obtain geoid separation from new geo coord service.
 
 ### iModel Transformations
 
-- Create examples and templates illustrating usage of iModel-to-iModel transformations and synchronization.
+- [ ] Create examples and templates illustrating usage of iModel-to-iModel transformations and synchronization.
```

Comment:
@MichaelBelousov do we have transformer examples or templates?

Reply:
- We have https://github.com/iTwin/etl-samples which I am cleaning up for 3.0
- Some of the tests are intended to be readable examples, but only 1 of them is exposed (using the doc generator's `[[include:file]]` functionality).
- I have a draft PR with documentation on examples for implementing branching with the transformer which I'm waiting for feedback from some people actively working on branching before I undraft

Reply:
If you expect those to be released roughly at the same time as 3.0, we can check the box now.

Reply:
sorry for the late reply. Yes, I just finished the ETL sample repo conversion, and the docs update will come out at the same time, can check this.